azure-pipelines.yml

```diff
@@ -7,58 +7,66 @@
 
 jobs:
 
-    - template: etc/ci/azure-linux.yml
-      parameters:
-          job_name: ubuntu16_cpython
-          image_name: ubuntu-16.04
-          python_versions: ['3.6', '3.7', '3.8', '3.9']
-          test_suites:
-              all: tmp/bin/pytest -vvs
-
-    - template: etc/ci/azure-linux.yml
+    - template: etc/ci/azure-posix.yml
       parameters:
           job_name: ubuntu18_cpython
           image_name: ubuntu-18.04
-          python_versions: ['3.6', '3.7', '3.8', '3.9']
+          python_versions: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Let's do 3.8+ actually, while we're at it.

Reply:
Done.